src/store/main/sagas/sweepQueryFlow.ts

```diff
@@ -27,19 +27,24 @@ export function* sweepQueryFlow() {
   const results: any[] = yield all(exchangeContractsAddresses.map(
     (addr) => call(queryFlows, addr),
   ));
-  const response: Unwrap<typeof queryStreams> = yield call(queryStreams, address);
+
   const streamedSoFarMap: { [key:string]: number } = {};
-  (response?.data?.data?.streams || [])
-    .forEach((stream:any) => {
-      const streamedSoFar = streamedSoFarMap[`${stream.token.id}-${stream.receiver.id}`] || 0;
-      Object.assign(streamedSoFarMap, {
-        [`${stream.token.id}-${stream.receiver.id}`]: Number(streamedSoFar) + Number(calculateStreamedSoFar(
-          stream.streamedUntilUpdatedAt,
-          stream.updatedAtTimestamp,
-          stream.currentFlowRate,
-        )),
+
+  if (address) {
```

Comment:
only query streams when any wallet is connected.

Reply:
I made a solution for this before I saw yours, which do you think is better

![image](https://user-images.githubusercontent.com/69639595/160527784-b641a160-109f-40e8-9008-ac6d63ac0856.png)

Reply:
Please make sure that it works and check your browser console, thanks!